qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/resolvers/OrValueResolverTest.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+* Copyright (c) 2022 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.resolvers;
+
+import static com.redhat.qute.QuteAssert.assertHover;
+import static com.redhat.qute.QuteAssert.c;
+import static com.redhat.qute.QuteAssert.r;
+import static com.redhat.qute.QuteAssert.testCompletionFor;
+import static com.redhat.qute.QuteAssert.testDiagnosticsFor;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for 'or' value resolver.
+ *
+ * @see https://quarkus.io/guides/qute-reference#built-in-resolvers
+ *
+ */
+public class OrValueResolverTest {
+
+	@Test
+	public void diagnostics() {
+		String template = "{@org.acme.Item item}\r\n" + //
+				"{item.or}";
+		testDiagnosticsFor(template);
+
+		template = "{@org.acme.Item item}\r\n" + //
+				" \r\n" + //
+				"{item.name ?: 'John'}";
+		testDiagnosticsFor(template);
+
+		template = "{@org.acme.Item item}\r\n" + //
+				" \r\n" + //
+				"{item.name.or('John')}";
+		testDiagnosticsFor(template);
```

Comment:
Here test failed although it should not.I think (but not sure), that https://github.com/redhat-developer/quarkus-ls/pull/516 could fix this problem.

Reply:
The stacktrace:
```
"The method `or(Object)` in the type `null` is not applicable for the arguments `(String)`."
```

Reply:
Indeed it's a bug and I hope #516  will fix it. Once @fbricon will merge the #516  PR please rebase your PR and tell me if it fixes the problem.